config/groups.yaml

```
@@ -32,6 +32,12 @@
     - "2509095"
     - "2553338"
   id: 189499
+  - name: "SSHiverMeTimbers"
+    members:
+    - "2585019"
+    - "2619820"
+    - "2614348"
+    - "2567906"
```

I think this has too much indentation now: 
```suggestion
- name: "SSHiverMeTimbers"
  members:
    - "2585019"
    - "2619820"
    - "2614348"
    - "2567906"
```